target/linux/ipq806x/config-6.1

```diff
@@ -35,9 +35,8 @@ CONFIG_ARM_ATAG_DTB_COMPAT_CMDLINE_MANGLE=y
 CONFIG_ARM_CPUIDLE=y
 CONFIG_ARM_CPU_SUSPEND=y
 # CONFIG_ARM_CPU_TOPOLOGY is not set
-CONFIG_ARM_CRYPTO=y
 CONFIG_ARM_GIC=y
-CONFIG_ARM_HAS_SG_CHAIN=y
+CONFIG_ARM_HAS_GROUP_RELOCS=y
```

Comment:
Do you really need these changes to kernel configuration for your device to work?
If this is needed please explain it in more detail in the commit message, if it is not needed remove these changes to the config-6.1 file please.

Reply:
Aw crap. Everytime I edit, it goes back to spaces unless I manually override.  

Fixed again.

Reply:
The code base is changing underneath me.  I will sync up again.

Reply:
If you device works fine without doing any changes to `target/linux/ipq806x/config-6.1` just do not do any changes to the file.

Reply:
Seems like all the differences come about after running "make kernel_menuconfig" to add CONFIG_NVMEM_U_BOOT_ENV.  Unless I'm doing something wrong, it seems like the previous config-6.1 is out of date with the latest kernel and no one has run kernel_menuconfig and pushed the result for a while.

Reply:
The configuration was probably not synced correctly before. If you only want to add `CONFIG_NVMEM_U_BOOT_ENV`, please only add the change adding this option.

Reply:
Please add only the changes you really need for this device to the config-6.1.